lab_2_tokenize_by_bpe/main.py

```diff
@@ -14,6 +14,17 @@ def prepare_word(
     :param end_of_word: a token that signifies the end of word
     :return: preprocessed word
     """
+    if not isinstance(raw_word, str) \
+            or not isinstance(start_of_word, (str | None)) \
+            or not isinstance(end_of_word, (str | None)):
+        return None
+    if start_of_word is None:
```

Comment:
too dificult, let's change the conditions

Reply:
fixed